### PR TITLE
MGMT-11729: Retrieve equinix metadata from rehearse jobs

### PR DESCRIPTION
**Pull request by adriengentil**
Compute the job context for rehearse jobs which makes possible to
download equinix metadata for them.

### CODE REVIEW

File: src/prowjobsscraper/prowjob.py [hunk 1]
```diff
@@ -92,15 +118,32 @@ def context(self) -> str:
...
+        ref = self.spec.extra_refs[0]
+        job_prefix = job_prefix + _JOB_PREFIX_TEMPLATE.format(
+            type=_TYPE_PREFIX[self.spec.type],
+            org=ref.org,
+            repo=ref.repo,
+            branch=ref.base_ref,
         )
-        return job_name
+        if self.metadata.labels.variant:
+            job_prefix = f"{job_prefix}{self.metadata.labels.variant}-"
```
Comment (adriengentil) [inline src/prowjobsscraper/prowjob.py:144]: TODO: I need to check that

**Review by osherdp — APPROVED** [review]

File: src/prowjobsscraper/prowjob.py (hunk 1 above)
Comment (adriengentil) [reply src/prowjobsscraper/prowjob.py:144]: all good, it should work :crossed_fingers:

**Pull request merged**